lib/govuk_content_schemas/frontend_schema_generator.rb

```diff
@@ -12,6 +12,7 @@ class GovukContentSchemas::FrontendSchemaGenerator
     redirects
     routes
     update_type
+    change_note
```

Comment:
> Is this correct for frontend? 

I believe this matches the behaviour of the publishing_api, the new top level `change_note` field is only sometimes used in the construction of the `change_history` member of the `details` object when presenting to the content store. These `INTERNAL_PROPERTIES` are members of the publisher schema to omit from the frontend schema.

> Shouldn't change notes be sent as an array in change history or similar?

Yes, as above, sometimes the Publishing API will construct the `change_history` from the change notes it has received.